templates/desktop/snappy.html

```diff
@@ -7,215 +7,176 @@
 
 {% block second_level_nav_items %}
```

Comment:
Page title should be `Snaps on Ubuntu` instead of `Installing snaps on ubuntu desktop`.